OrbitClientGgp/ClientGgp.cpp

```diff
@@ -235,11 +234,12 @@ void ClientGgp::ProcessTimer(const TimerInfo& timer_info) { timer_infos_.push_ba
 
 // CaptureListener implementation
 void ClientGgp::OnCaptureStarted(
-    int32_t process_id, std::string process_name, std::shared_ptr<Process> process,
+    std::unique_ptr<ProcessData> process,
+    absl::flat_hash_map<std::string, ModuleData*>&& module_map,
```

Comment:
This does not look right. The module map should probably get updated from the service at the start capture - seems like it should be a separate event in the listener. Now we kind of assume that whatever is happened to be on the client at the time of start is the correct module map, which is not necessarily the case.

(not for this PR - but maybe add a todo here and a bug about this.)

Reply:
I see and agree. I filed a bug and made a TODO comment in CaptureClient